app/components/article-featured-video.js

```diff
@@ -43,7 +46,7 @@ export default Component.extend(
 		 */
 		initOnScrollBehaviour() {
 			const $video = this.$('.article-featured-video__container'),
-				videoHeight = $video.height(),
+				videoHeight = this.get('withinPortableInfobox') ? 97 : 230,
```

Comment:
it happened few times that I got 0 as a height taken from DOM, and since video hight is constant this in my opinion is simplest solution